src/playground/addons/story-utils/index.js

```diff
@@ -649,3 +649,10 @@ export const getFormControls = (data, type) => {
 
   return argTypes;
 };
+
+export const getOptions = (data) => {
```

Comment:
The name of the utility is not describing it's purpose quite exactly. The function converts an array of strings to an object of key value pairs containing the the same strings. I'm not sure whether other cases will interest of the same conversion on the level of making an options list of just any contol.

As suggested in https://github.com/ec-europa/europa-component-library/pull/1855/files#r547903798 we can import  uiIcons directly in this story-utils file, prepare the list of icons and use them in a contol-related utility. If relevant the list of icons can also be exported, but i doubt it will be useful

This file 
```javaascript
import uiIcons from '@ecl/resources-ec-icons/dist/lists/ui.json';

export const iconList = uiIcons.reduce((a, b) => {
    a[b] = b;
    return a;
  }, {});
  
  export const getIconControls = (data) => {
//  return the contols structure
  }
```

Stories will then be able to pick whichever way `import { iconList } from '@ecl/story-utils';` for the ready list or `import { getIconControls } from '@ecl/story-utils';` for the ready control.

Reply:
Well, the idea was to provide a generic utility for creating the right object to pass to a select control, but the reality is that this is not needed, this was a legacy from the knobs, in controls i see we can pass directly the object as we import it, so i'm simply removing the utility and just using the imported list as it is.
Good if we have a utility for quickly importing icons knob but it would not be used i think in this case, here we just want to show that the icon can be chosen, not worth i think to have all the related controls.